ansible/roles/sufia/tasks/main.yml

```diff
@@ -89,6 +89,8 @@
       mode: "0770"
     - path: "{{ project_app_root }}/db"
       mode: "0770"
+    - path: "{{ project_app_root }}/public/uploads"
```

Comment:
Even with the tmp/uploads path being assigned to the upload_path variable the application is still trying to write files to public/uploads. We've yet to figure out why.